definitions/json/CaseEvent/CaseEvent-ET1Repped.json

```
@@ -70,8 +73,10 @@
     "CallBackURLSubmittedEvent": "${ET_COS_URL}/et1Repped/submitted",
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
-    "ShowSummary": "N",
     "EndButtonLabel": "Submit ET1",
+    "ShowSummary": "N",
+    "SignificantEvent": "Yes",
+    "TTLIncrement": "365",
```

This isn't needed as this submits the draft case into a live case
```suggestion
    "ShowSummary": "N"
```